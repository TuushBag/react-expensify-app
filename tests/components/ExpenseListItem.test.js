import React from 'react'
import { shallow } from 'enzyme'
import ExpenseListItem from '../../src/components/ExpenseListItem'
import expenses from '../fixtures/expenseData'


test('should test snapshot of expenselistitem', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
    expect(wrapper).toMatchSnapshot()
})