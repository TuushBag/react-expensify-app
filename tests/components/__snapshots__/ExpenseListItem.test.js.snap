// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test snapshot of expenselistitem 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h2>
      Internet
    </h2>
  </Link>
  <p>
    1293
     - 
    0
  </p>
</div>
`;
